apps/web/app/(app)/environments/[environmentId]/project/integrations/notion/components/AddIntegrationModal.tsx

```
@@ -38,6 +39,59 @@ import {
+      case ERRORS.UNSUPPORTED_TYPE:
+        return (
+          <>
+            -{" "}
+            {t("environments.integrations.notion.col_name_of_type_is_not_supported", {
+              col_name: col.name,
+              type: col.type,
+            })}
+          </>
+        );
+      case ERRORS.MAPPING:
+        const element = getElementTypes(t).find((et) => et.id === elem.type);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)